examples/objective-c-ios/Bugsnag Test App.xcodeproj/project.pbxproj

```diff
@@ -123,16 +111,6 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
-		8ACFFA0A1D895D4D00357B5E /* custom-keyboard */ = {
```

Comment:
This seems to have been removed from the example but is still present as a scheme in XCode?